tools/events/CMakeLists.txt

```diff
@@ -1,3 +1,18 @@
-include_directories(./ inc/)
-add_library(ion_events ion_event_util.cpp ion_event_stream.cpp ion_event_equivalence.cpp)
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+add_library(ion_events
+        ion_event_util.cpp
+        ion_event_stream.cpp
+        ion_event_equivalence.cpp)
+
+target_include_directories(ion_events
+        PUBLIC
+            ${CMAKE_CURRENT_SOURCE_DIR}/inc
```

Comment:
should this be "include"?

Reply:
This points to the subfolder for `event` which is only used internally in our code. I am not 100% sure we want this lib exposed to customers.

Reply:
Yes, let's not expose the stuff in `events` right now.